test/e2e/performanceprofile/functests/1_performance/rt-kernel.go

```diff
@@ -71,4 +76,25 @@ var _ = Describe("[performance]RT Kernel", func() {
 		err = nodes.HasPreemptRTKernel(&nonPerformancesWorkers[0])
 		Expect(err).To(HaveOccurred(), "Node should have non-RT kernel")
 	})
+
+	It("[test_id:rctree] Should have kernel param rcutree.kthread", func() {
```

Comment:
Here we have another test to check kernel boot args:
https://github.com/openshift/cluster-node-tuning-operator/blob/a7fcf60362a2dd11519055d865df1782263854ff/test/e2e/pao/functests/1_performance/performance.go#L122

It is necessary to do it in a new file? Could it be added there?

Reply:
yes np, will do in the PR targeting master branch.